snapd-glib/snapd-autorefresh-change-data.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2024 Canonical Ltd.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2 or version 3 of the License.
+ * See http://www.gnu.org/copyleft/lgpl.html the full text of the license.
+ */
+
+#include "snapd-autorefresh-change-data.h"
+
+struct _SnapdAutorefreshChangeData {
+    SnapdChangeData parent_instance;
+    GStrv snap_names;
+    GStrv refresh_forced;
+};
+
+G_DEFINE_TYPE (SnapdAutorefreshChangeData, snapd_autorefresh_change_data, SNAPD_TYPE_CHANGE_DATA)
+
+enum
+{
+    PROP_SNAP_NAMES = 1,
+    PROP_REFRESH_FORCED,
+    N_PROPERTIES
+};
+
+static GParamSpec *obj_properties[N_PROPERTIES] = { NULL, };
```

Comment:
Nit: this doesn't match the property style of the other objects - just call `g_object_class_install_property` for each property and this array is unnecessary.

Reply:
Done.